foundations/javascript_basics/variables_and_operators.md

```diff
@@ -1,86 +1,128 @@
 
 ```html
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Page Title</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
 </head>
 <body>
+
   <script>
     // Your JavaScript goes here!
     console.log("Hello, World!")
   </script>
+
 </body>
 </html>
 ```
 
-Save and open this file up in a web browser (you can use ["Live Server" on Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) to do this!) and then <span id="access-devTools-console">open up the browser's console by right-clicking on the blank webpage and selecting "Inspect" or "Inspect Element".  In the 'Developer Tools' pane find and select the 'Console' tab</span>, where you should see the output of our `console.log` statement.
+Save and open this file up in a web browser and then open up the browser's console:
 
-> <span id="console-log">`console.log()` is the command to print something to the developer console in your browser. You can use this to print the results from any of the following articles and exercises to the console.</span> We encourage you to code along with all of the examples in this and future lessons.
+1. Right-click on the blank webpage.
+1. Click on "Inspect" or "Inspect Element" to open the Developer Tools.
+1. Find and select the "Console" tab, where you should see the output of our `console.log` statement.
+
+**Tip:** You can use ["Live Server" extension in Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) to automatically update the browser when you save your file instead of having to manually refresh the page to see any changes when you edit your code. Try edit the text to say something different!
+
+`console.log()` is the command to print something to the developer console in your browser. You can use this to print the results from any of the following articles and exercises to the console. We encourage you to code along with all of the examples in this and future lessons.
 
 Another way to include JavaScript in a webpage is through an external script. This is very similar to linking external CSS docs to your website.
 
 ```html
-  <script src="javascript.js"></script>
+<script src="javascript.js"></script>
 ```
 
 JavaScript files have the extension `.js` similar to `.css` for stylesheets. External JavaScript files are used for more complex scripts.
 
+We named our file `javascript.js` but we could have chosen any name like `my-script.js` or even no name `.js`. What is really important is the `.js` extension.
+
 ### Variables
 
+These are the building blocks of any program. As an abstract example, let's consider a program that takes in two numbers and returns the sum of those numbers.
+
+It could store the first number as a variable, `a` and the second number as another variable `b`. Then it would output their sum - `a + b`.
+
```

Comment:
I'm not sure starting with an example like this makes the concept less confusing. I'd take just the first sentence here, then maybe do the example after the illustration link below.

Reply:
Ok, so that makes sense. I've removed the example since I don't think it really fits. The example with adding two variables is unnecessary (learners may not even be fully comfortable with first concept, but we are already introducing another one?)